src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -1946,6 +1950,32 @@ abstract class DatabaseTest {
         )
 
     cohortModulesDao.insert(row)
+
+  fun insertVote(
+      projectId: ProjectId,
+      phase: CohortPhase = CohortPhase.Phase0DueDiligence,
+      user: UserId = currentUser().userId,
+      voteOption: VoteOption? = null,
+      conditionalInfo: String? = null,
+      createdBy: UserId = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
+  ): ProjectVotesRow {
+    val row =
+        ProjectVotesRow(
+            createdBy = createdBy.toIdWrapper { UserId(it) },
```

Comment:
Since the user IDs here are already of type `UserId`, there's no need to call `toIdWrapper`; this can just be `createdBy = createdBy`. (And it's fine for `createdBy` to not be of type `Any` here since tests are unlikely to want to explicitly pass in specific user ID values.)